e2e/helpers/docker_compose.py

```diff
@@ -10,7 +10,7 @@ def stop():
 
 def clean():
     stop()
-    subprocess.check_call("docker-compose rm -vf".split(' '))
+    subprocess.check_call("docker-compose rm -f".split(' '))
```

Comment:
If we don't rely on volumes for the caching, this change isn't needed for the stated purpose of this pull request. (Maybe it's a good idea anyway, though.)